webhook/batch_url_notifier.go

```diff
@@ -0,0 +1,155 @@
+package webhook
```

Comment:
Write tests for this new notifier. Maybe assert that all queued events has been received.